### PR TITLE
Poisson reconstruction: Provide a simple function

### PR DESCRIPTION
**Pull request by sgiraudot**
## Summary of Changes

This adds a syntaxic sugar simple function for the Poisson reconstruction algorithm (which, at the moment, is pretty complicated to use as it lets the user do everything after computing the implicit function). This is a small feature.

## Release Management

* Affected package(s): Poisson Surface Reconstruction
* Issue(s) solved (if any): fix https://github.com/CGAL/cgal/issues/1659
* [Small Features/Provide_simple_function_for_Poisson_Reconstruction](https://cgal.geometryfactory.com/CGAL/Members/wiki/Features/Small_Features/Provide_simple_function_for_Poisson_Reconstruction) (pre-approved by Pierre Alliez, 2017/06/12)

### CODE REVIEW

File: Installation/changes.html [hunk 1]
```diff
@@ -260,6 +260,13 @@ <h2 id="release4.10">Release 4.10 </h2>
 
 
 <!-- Surface Reconstruction -->
+  <h3>Poisson Surface Reconstruction</h3>
+  <ul>
+    <li>A new global
+    function <code>CGAL::poisson_surface_reconstruction_delaunay()</code>
+    is provided in addition to the current class-based API in order to
+    make it easier to use.</li>
+  </ul>
```
Comment (lrineau) [inline Installation/changes.html]: That branch seems rather old, and this section of the changelog is in the changes of CGAL-4.10!

**Comment by sgiraudot** [comment]
I rebased the branch and fix the `changes.html` issue.

**Comment by lrineau** [comment]
There is still a compilation error detected by Travis: https://travis-ci.org/CGAL/cgal/jobs/241895078#L4106

**Comment by sgiraudot** [comment]
It should be fixed now, let's see.

**Pull request merged**